massa-client/src/cmds.rs

```diff
@@ -424,8 +425,15 @@ impl Command {
 
     /// Returns true if the command needs wallet access
     pub(crate) fn is_pwd_needed(&self) -> bool {
-        !(self.get_str("pwd_not_needed").is_some()
-            && self.get_str("pwd_not_needed").unwrap() == "true")
+        let pwd_not_needed = self.get_str("pwd_not_needed");
+        !pwd_not_needed.map_or(false, |f| f == "true")
```

Comment:
Thank you for the double negation x)

Maybe a match is more readable? (but it's only a suggestion)
```suggestion
    match self.get_str("pwd_not_needed") {
        Some("true") => false, // pwd_not_needed si "true", so return false
        _ => true,            // Else, pwd is needed
    }
```